libcore/src/math/VelocityModel.cpp

```diff
@@ -75,14 +76,14 @@ bool VelocityModel::Init(Building * building)
         int res = ped->FindRoute();
         if(!ped_is_waiting && res == -1) {
             std::cout << ped->GetID() << " has no route\n";
-            Log->Write(
-                "ERROR:\tVelocityModel::Init() cannot initialise route. ped %d is deleted in Room "
+            LOG_ERROR(
+                "VelocityModel::Init() cannot initialise route. ped {:d} is deleted in Room "
                 "%d %d.\n",
                 ped->GetID(),
                 ped->GetRoomID(),
                 ped->GetSubRoomID());
             building->DeletePedestrian(ped);
-            Log->incrementDeletedAgents();
+            // TODO KKZ track deleted peds
```

Comment:
:+1

Reply:
Ah i should have written about that in the PR description: I think we should add a `stats` object to keep track of things like this but I wanted to keep this out if this PR

Reply:
Now tracked in  #627